CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 * Fix: dial new peers on CSRs replication
 
+* Feature: add functionality to export chat to text document in desktop version
```

Comment:
Could you add this info under [unreleased] on top of the changelog? I think first you have to merge develop into your branch because we released a new alpha version today

Reply:
Oh, okay! I merged the develop branch just now. I'll check once more.

Reply:
@EmiM done!